pkgs/by-name/um/umu-launcher-unwrapped/makefile-installer-prefix.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/Makefile.in b/Makefile.in
+index b82053d..3e4379e 100644
+--- a/Makefile.in
++++ b/Makefile.in
+@@ -90,7 +90,7 @@ umu-dist: $(OBJDIR)/.build-umu-dist
+ umu-dist-install: umu-dist
+ 	$(info :: Installing umu )
+ 	install -d $(DESTDIR)$(PYTHONDIR)/$(INSTALLDIR)
+-	$(PYTHON_INTERPRETER)  -m installer --destdir=$(DESTDIR) $(OBJDIR)/*.whl
++	$(PYTHON_INTERPRETER)  -m installer --prefix=$(DESTDIR) $(OBJDIR)/*.whl
```

Comment:
Can this be upstreamed, or does it only make sense in nixpkgs?

Reply:
Probably not upstreamed, as it changes the behavior significantly, and I also don't want to mess with python build tooling more than I have to; but if someone has a better solution, then I can change it.

Reply:
Believe this is for Open-Wine-Components/umu-launcher#296 ? We fix this upstream with a patch several dependencies are vendored